### PR TITLE
Cole/BFB-310: Added Water Shader

### PR DESCRIPTION
**Pull request by csweet22**
**JIRA Ticket**
https://perigongames.atlassian.net/browse/BFB-310

**Description**
Added in a water shader under **Assets > _BForBoss > _LevelDesign > Materials > Water**

Example of the water can be seen in MT_Weapon_Sandbox.

Material has many different options to customize it.

I'd recommend using ProBuilder to create water planes. You can also choose the resolution of the plane (height and width cuts) to add resolution for the position displacement.

Future TODO: Consider adding a distortion effect to the normal maps.

**Screenshots / GIFs**

image

### CODE REVIEW

**Comment by m4rtint** [comment]
Are there currently any parameters to tweek?

**Comment by csweet22** [comment]
> Are there currently any parameters to tweek?

Yup there are quite a few! They can be found in the Inspector under the Material Shader.

File: .gitignore [hunk 1]
```diff
@@ -91,3 +91,4 @@ FMODAssets/.user/
 # If the source bank files are kept outside of the StreamingAssets folder then these can be ignored.
 # Log files can be ignored.
 fmod_editor.log
+Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset
```
Comment (m4rtint) [inline .gitignore:94]: https://qa.fmod.com/t/fmodstudiosettings-asset-keeps-appearing-in-git-without-changing-anything/17318

**Comment by csweet22** [comment]
FMOD asset is back in! Let me know if there is anything else that needs altering 👍

**Review by m4rtint — APPROVED** [review]

**Pull request merged**